ros_buildfarm/binarydeb_job.py

```diff
@@ -86,7 +88,18 @@ def build_binarydeb(rosdistro_name, package_name, sourcedeb_dir):
 
     cmd = ['apt-src', 'build', source]
     print("Invoking '%s' in '%s'" % (' '.join(cmd), source_dir))
-    subprocess.check_call(cmd, cwd=source_dir)
+    try:
+        subprocess.check_call(cmd, cwd=source_dir)
+    except subprocess.CalledProcessError:
+        traceback.print_exc()
+        sys.exit("""
+--------------------------------------------------------------------------------------------------------
+`{0}` failed.
+This is usually because of an error building the package.
+The traceback from this failure (just above) is printed for completeness, but you can usually ignore it.
+You should look above `E: Building failed` in the build log to look for the actual cause of the failure.
```

Comment:
Yeah, I reworded it a few times and left the extra `to look` in there. Fixed in edc3349